zwave_js_server/client.py

```diff
@@ -102,6 +106,7 @@ async def connect(self) -> None:
             self._client = await self.aiohttp_session.ws_connect(
                 self.ws_server_url,
                 heartbeat=55,
+                max_msg_size=MAX_NETWORK_STATE_SIZE,
```

Comment:
Why add it here? Don't we only need it in `dump.py` ?

Reply:
No, it's on any client connection.

Reply:
see https://github.com/home-assistant-libs/zwave-js-server-python/pull/303#issuecomment-932694444